java/cdk/src/main/java/sleeper/cdk/stack/QueryStack.java

```diff
@@ -128,71 +133,191 @@ public QueryStack(Construct scope,
 
         instanceProperties.set(QUERY_TRACKER_TABLE_NAME, queryTrackingTable.getTableName());
 
-        // Queue for queries
+        // Queue for queries register
+        Queue queryRegisterQueue = setupQueryRegisterQueue(instanceProperties);
+
+        // Queue for leaf partition queries
+        Queue leafPartitionQueriesQueue = setupLeafPartitionQueryQueues(instanceProperties);
```

Comment:
It might be easier to read here if you group the queryRegister and leafPartitionQuery infrastructure together.

You could have a method which creates the queryRegisterLambda, and a method which creates the leafPartitionQueryLambda, and move a lot of the code out of this constructor. I was thinking in the end the constructor would look something like this:
```java
LambdaCode queryJar = jars.lambdaCode(BuiltJar.QUERY, jarsBucket);
Queue queryRegisterQueue = setupQueryRegisterQueue(instanceProperties);
IFunction queryRegisterLambda = setupQueryRegisterLambda(...);

Queue leafPartitionQueriesQueue = setupLeafPartitionQueryQueues(instanceProperties);
IFunction leafPartitionQueryLambda = setupLeafPartitionQueryLambda(...);

this.setupWebSocketApi(queryJar, instanceProperties, queryRegisterQueue, queryRegisterLambda, coreStacks);
Utils.addStackTagIfSet(this, instanceProperties);
```

Reply:
I have refactored the class and the constructor is a lot smaller